Tests/LoggerTests.swift

```diff
@@ -296,6 +296,81 @@ class SynchronousLoggerLogLevelTestCase: SynchronousLoggerTestCase {
         // Then
         XCTAssertEqual(writer.actualNumberOfWrites, 10, "Actual number of writes should be 10")
     }
+
+    func testThatItLogsAsExpectedWithAutoclosureDebugLogLevel() {
+        // Given
+        let (log, writer) = logger(logLevel: .Debug)
+
+        // When
+        log.debug ( "" )
```

Comment:
given that these are standard function syntax now, can we update these to not include the extra spaces?

```
log.debug("")
log.info("")
etc...
```